### PR TITLE
WD-7565 LXD install page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Build LXD install page

## QA

- Check https://canonical-com-1127.demos.haus/lxd/install against [copy docs](https://docs.google.com/document/d/1LKlXAc39dy69dSCc7PE6WqV2qkCSeXmicKGS4lFicDE/edit) and [design](https://www.figma.com/file/ZU2ufsTJD9tEkXPqhhDMZz/U.com---LXD-(rebranding)?node-id=447%3A1412&mode=dev)
Note: visual designer has changed the original layout that came from the agency, so please compare it loosely.

- Click on "Get in touch" and test that it submits successfully (you can fill it up with "test" for all fields, it'll be ignored in marketo)
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue

Fixes https://warthogs.atlassian.net/browse/WD-7565

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

File: templates/lxd/_form-lxd-install.html
Comment (anthonydillon) [inline templates/lxd/_form-lxd-install.html:9]:
```suggestion
          <button class="p-modal__close p-modal-close-button u-no-margin--top" aria-controls="lxd-install-modal" aria-label="Close active modal">Close</button>
```

File: templates/lxd/index.html
Comment (anthonydillon) [inline templates/lxd/index.html:9]: Should this be removed?

File: templates/lxd/index.html
Comment (anthonydillon) [inline templates/lxd/index.html:41]: Tabbing is off here

File: templates/lxd/index.html
Comment (anthonydillon) [inline templates/lxd/index.html:271]: This is the wrong link

File: templates/lxd/index.html
Comment (anthonydillon) [inline templates/lxd/index.html:282]:
```suggestion
  {% set returnURL = "https://canonical.com/lxd#success" %}
```

File: templates/lxd/index.html
Comment (anthonydillon) [inline templates/lxd/index.html:283]:
```suggestion
  {% include "lxd/_form-lxd-install.html" %}
```

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:26]: Does this page exist?

**Review by anthonydillon — COMMENTED** [review]
Some quick comments in the code.

**Comment by Sophie-32** [comment]
added some comments to copydoc, and one comment in Figma for @lyubomir-popov to see if we have time to make a change.

**Comment by carkod** [comment]
@lyubomir-popov MM comments addressed, can I get another review?

**Comment by lyubomir-popov** [comment]
@carkod I reviwed and pushed a commit fixing a few small things. The layout has changed in a couple of places, it works better that way in my opinion.

**Comment by carkod** [comment]
@Sophie-32 can I get another review when you can?

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:85]: You can't wrap `pre` in a `p`. The browser will assume an error and auto code the p which leads to invalid markup and unexpected space generated with multiple empty p tags in the final rendered mark-up.

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:102]: This example is missing the the command flag to work. `--vm`

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:100]: Personally I feel this is a strange way to show an example. It looks like two commands at first.

File: static/sass/styles.scss
Comment (anthonydillon) [inline static/sass/styles.scss:835]: Looks like your not using this is the markup. Can it be removed?

File: templates/lxd/_form-lxd-install.html
Comment (anthonydillon) [inline templates/lxd/_form-lxd-install.html:22]: There are a number of fields here with duplicates required attributes.

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:228]: This is the wrong returnURL

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:227]: This form has 4446 as the id on the form element

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:143]: This entire sentence should be the link.

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:135]: These shallow sections sure make the render on the phone look strange. There is large gaps between the heading and content.

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:203]:
```suggestion
          <p>We welcome and value contributions from the community.</p>
```

File: templates/lxd/install.html
Comment (anthonydillon) [inline templates/lxd/install.html:218]: This links to a root level contract us page. It's that correct?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
A few comments in the code and some content suggestions in the copy doc.

File: templates/lxd/_form-lxd-install.html
Comment (anthonydillon) [inline templates/lxd/_form-lxd-install.html:10]: Duplicated `id` attr.

File: templates/lxd/install.html
Comment (carkod) [reply templates/lxd/install.html:143]: We usually don't make entire sentences a link, so that it can create emphasis, I believe this is the usual practice we follow?

File: templates/lxd/install.html
Comment (carkod) [reply templates/lxd/install.html:218]: That's what we do on all modal forms on c.com. The link triggers a modal already, so I don't see the need to create a separate form that looks the same as the modal form.

File: templates/lxd/install.html
Comment (carkod) [reply templates/lxd/install.html:125]: It's up to @lyubomir-popov  but I think we have this spacing consistently in all headings, in which case I don't think we should change it?

File: templates/lxd/install.html
Comment (carkod) [reply templates/lxd/install.html]: Done, thanks

File: static/sass/styles.scss
Comment (carkod) [reply static/sass/styles.scss]: Removed.

File: templates/lxd/_form-lxd-install.html
Comment (carkod) [reply templates/lxd/_form-lxd-install.html]: Fixed.

File: templates/lxd/install.html
Comment (akbarkz) [reply templates/lxd/install.html]: Nevertheless, I see it has been fixed

**Review by akbarkz — APPROVED** [review]

**Pull request merged**
